BEP-0003 - noCorrelation relationship

ncatlett opened the pull request:
add noCorrelation relationship

ncatlett committed:
start BEP-0003

add noCorrelation relationship

ncatlett commented:
I followed the directions in the tutorial PDF for making the pull request for a new BEP - I think I may have deleted the template.

wshayes committed:
Replaced template

wshayes commented:
I like the relationship proposal and like how it handles ambiguous correlations, but I prefer to not have an abbreviation for it - at least 'no' doesn't work for me when reading through a sample relationship.

ncatlett commented:
agree that 'no' does not work as an abbreviation

cthoyt commented:
Also agree that `no` is a bad abbreviation. I think `cor` is the abbreviation for the non-directional `correlation` relationship, so maybe `noCor`? 

------------
What do you guys think about the idea of "negative" knowledge as a more general concept?

There's already the `causesNoChange` relationship, which is effectively the negative of `regulates`. `noCorrelation` would be the negative of the `correlation` relationship. 

I think it's the case that there are a lot of times there is a "negative" version of a statement (even direct and causal relations) in the literature, but we don't curate it because it's just not that useful. BEL isn't always oriented towards utility, but rather just encoding what's possible. Conversely, I would imagine representing negative knowledge in a network would be difficult, and so would reasoning over it.

ncatlett committed:
Update bep-0003.md

Added sentences explicitly stating that noCorrelation is non-directional like all other correlative relationships

wshayes commented:
Approved in a vote by the BEP Review team - John Bachman, Charlie Hoyt and William Hayes.
